design-guidelines/interactive-element-states.md

```diff
@@ -11,7 +11,33 @@ States communicate the different statuses of UI components to the user, includin
 - Data inputs
   - For example: a text input
 
-## States
```

Comment:
> which component's or patterns this state does not apply to.

The list at the top addresses what kind of components we're talking about: https://github.com/dcos-labs/ui-kit/blob/67d8afc93054dcd0e120e3b0e9e30d5427e25d3b/design-guidelines/interactive-element-states.md#common-interaction-subjects.

I don't think we should reference specific components, because these guidelines are meant to be general. Component-specific docs live in Storybook.